src/cbc_sdk/platform/observations.py

```diff
@@ -601,3 +610,52 @@ def __getitem__(self, item):
         else:
             raise AttributeError("'{0}' object has no attribute '{1}'".format(self.__class__.__name__,
                                                                               item))
+
+
+class NetworkThreatMetadata(NewBaseModel):
+    """Represents an NetworkThreatMetadata"""
```

Comment:
should be "a Network..." instead of "an ..."

Reply:
Fixed